pkg/fixtures/fixtures.go

```diff
@@ -269,18 +269,52 @@ func (fxt *Fixture) parseArray(params []interface{}, parent string, index int) [
 
 func (fxt *Fixture) parseQuery(value string) string {
 	// Queries to fill data will start with #$ and contain a : -- search for both
-	// to make sure that we're trying to parse a query
-	r := regexp.MustCompile(`\${(.+):(.+)}`)
+	// to make sure that we're trying to parse a query.
+	// Additionally look for an optional default value: ${name:json_path|default_value}
+	r := regexp.MustCompile(`\${([^\|]+):([^\|]+)\|?(.+)?}`)
 	if r.Match([]byte(value)) {
 		nameAndQuery := r.FindStringSubmatch(value)
 		name := nameAndQuery[1]
 
+		// Check if there is a default value specified
+		if nameAndQuery[3] != "" {
+			value = nameAndQuery[3]
+		}
+
+		// Catch and insert .env values
+		if name == ".env" {
```

Comment:
Are there any unit tests you could add for this? It'd be helpful to verify that the parsing works and we don't accidentally introduce regressions. Check out https://github.com/stripe/stripe-cli/blob/master/pkg/fixtures/fixtures_test.go